config.example.yaml

```diff
@@ -213,6 +225,9 @@ services:
       salt: secret
       default_page_size: 250
       user_agent: RESTBase
+      internal_request_whitelist:
```

Comment:
@gwicke I don't agree with your understanding of swagger doc wording.

[Here](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#securitySchemeObject) validity column is related to the type of security definition object where the particular field can be added (e.g. `in` field is only valid in an object of `apiKey` type). To support my understanding [here](https://github.com/swagger-api/swagger-spec/blob/master/schemas/v2.0/schema.json#L1226) is the complete JSON schema of the security definition object. Please look at each object type definition: the `type` field there is restricted to a single-value enum and only possible values are `basic`, `apiKey` and `oath2`, so if we say that `type` is `mediawiki` we would get an invalid spec.